sdk/textanalytics/Azure.AI.TextAnalytics/src/SentenceOpinion.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics.Models
+{
+    [CodeGenModel("SentenceOpinion")]
+    internal partial class SentenceOpinion
```

Comment:
Transform from enum `SentenceOpinionSentiment` to string so we can parse it into a `TextSentiment`